More/Logfile Analyzer.html

```diff
@@ -3,9 +3,13 @@
 	<head>
 		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 		<title>KTANE Logfile Analyzer</title>
-		<script src="../HTML/js/jquery.3.1.1.min.js"></script>
+		<script src="TDSBombRenderer/jquery.min.js"></script> <!-- Replacing the old jQuery with this newer version; the older one broke the TDSBombRenderer completely -->
+		<script src="TDSBombRenderer/anime.min.js"></script>
+		<script src="TDSBombRenderer/anime.min.js"></script>
```

Comment:
Is this intended to have duplicate entries?

Reply:
'Twas not intended, but now is fixed